app/server/account_management/account_management_app.js

```diff
@@ -3,16 +3,30 @@ const express = require("express");
 const app = express();
 const bodyParser = require("body-parser");
 const cors = require("cors");
+const mongoose = require("mongoose");
 
 const userManagementRouter = require("./routes/user_management_routes");
 
+
+
 // solves the CORS issue with the front end
 app.use(cors());
 // allows application accept json
 app.use(bodyParser.json()); 
 // allows application accept urlencoded post data
 app.use(bodyParser.urlencoded({ extended: true }));
 
+
+//Attempt to connect to MongoDB Atlas
+mongoose.connect(
+    "mongodb+srv://mindjogg_the_app:Mindj0Gg123@mindjoggcluster.e2dzp.mongodb.net/myFirstDatabase?retryWrites=true&w=majority",
```

Comment:
Is there any possibility of this URL changing? If so we should store it in a constant, if not then no worries